alert/store.go

```diff
@@ -202,6 +205,21 @@ func NewStore(ctx context.Context, db *sql.DB, logDB *alertlog.Store) (*Store, e
 			FROM services
 			WHERE id = $1
 		`),
+
+		feedback: p(`
```

Comment:
Can we use `sqlc` for this since it's new code?

Reply:
Yeah sorry, this is what's most familiar to me. Can do.